python/astro_metadata_translator/observationInfo.py

```diff
@@ -296,23 +367,32 @@ def __gt__(self, other):
     def __getstate__(self):
         """Get pickleable state
 
-        Returns the properties, the name of the translator, and the
-        cards that were used.  Does not return the full header.
+        Returns the properties.  Deliberately does not preserve the full
+        current state; in particular, does not return the full header or
+        translator.
 
         Returns
         -------
-        state : `dict`
-            Dict containing items that can be persisted.
+        state : `tuple`
+            Pickled state.
         """
         state = dict()
-        for p in self._PROPERTIES:
-            property = f"_{p}"
-            state[p] = getattr(self, property)
+        for p in self.all_properties:
+            state[p] = getattr(self, p)
 
-        return state
+        return self.extensions, state
```

Comment:
Does this mean we can't read a previously pickled ObservationInfo?

Reply:
That's right. Do you have reason to think these exist in the wild, or are you simply prompting me to document this change in behaviour somewhere?

Reply:
I'm not really sure but it is a change in interface and the package has been around for a while. Presumably we can see what state there is in the pickle and treat the extensions as optional?